### PR TITLE
docs: rename queue management topic

### PR DESCRIPTION
**Pull request by gh-determined-ai**
## Description

Renamed Queue Management topic to Job Queue Management.

## Test Plan

Describe the situations in which you've tested your change, and/or a screenshot as appropriate.
Reviewers may ask questions about this test plan to ensure adequate manual coverage of changes.
-->

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

**Comment by gh-determined-ai** [comment]
In that case, let's do a full copy edit. @hamidzr @eecsliu, Please address the TBDs in the latest push, if possible.

File: docs/features/job-queue.txt [hunk 1]
```diff
@@ -1,103 +1,159 @@
...
-The WebUI and the CLI display a table of results, ordered by scheduling order. The scheduling order
-does not represent the job priority. In addition to job order, the table includes the job states and
-number of slots allocated to each job.
+The WebUI and the CLI display a results table ordered by scheduling order. The scheduling order does
+not represent the job priority. In addition to job order, the listing shows:
+
++---------------+------------------------------------------------------------------------------+
+| Job Property  | Description                                                                  |
++===============+==============================================================================+
+| ``ID``        | TBD                                                                          |
```
Comment (eecsliu) [inline docs/features/job-queue.txt:90]: The unique ID assigned to the Job.

File: docs/features/job-queue.txt [hunk 2]
```diff
@@ -1,103 +1,159 @@
...
++---------------+------------------------------------------------------------------------------+
+| ``Type``      | TBD                                                                          |
```
Comment (eecsliu) [inline docs/features/job-queue.txt:92]: `TYPE_EXPERIMENT`, `TYPE_COMMAND`, `TYPE_NOTEBOOK`, `TYPE_SHELL`, or `TYPE_TENSORBOARD`.

File: docs/features/job-queue.txt [hunk 3]
```diff
@@ -1,103 +1,159 @@
...
++---------------+------------------------------------------------------------------------------+
+| ``Job Name``  | TBD                                                                          |
```
Comment (eecsliu) [inline docs/features/job-queue.txt:94]: Same as the display name of the experiment or command

File: docs/features/job-queue.txt [hunk 4]
```diff
@@ -1,103 +1,159 @@
...
++---------------+------------------------------------------------------------------------------+
+| ``Priority``  | TBD                                                                          |
```
Comment (eecsliu) [inline docs/features/job-queue.txt:96]: The latest priority value assigned to the job.

File: docs/features/job-queue.txt [hunk 5]
```diff
@@ -1,103 +1,159 @@
...
++---------------+------------------------------------------------------------------------------+
+| ``Submitted`` | TBD                                                                          |
```
Comment (eecsliu) [inline docs/features/job-queue.txt:98]: Job submission timestamp

File: docs/features/job-queue.txt [hunk 6]
```diff
@@ -1,103 +1,159 @@
...
++---------------+------------------------------------------------------------------------------+
+| ``Slots``     | (acquired/needed)                                                            |
++---------------+------------------------------------------------------------------------------+
+| ``Status``    | TBD                                                                          |
```
Comment (eecsliu) [inline docs/features/job-queue.txt:102]: `Queued` or `Scheduled`

File: docs/features/job-queue.txt [hunk 7]
```diff
@@ -1,103 +1,159 @@
...
++---------------+------------------------------------------------------------------------------+
+| ``User``      | TBD                                                                          |
```
Comment (eecsliu) [inline docs/features/job-queue.txt:104]: Owner of the job

File: docs/features/job-queue.txt (hunk 1 above)
Comment (eecsliu) [reply docs/features/job-queue.txt:90]: Not super sure if these should be full sentences or as succinct as possible, but hopefully these are ok.

File: docs/features/job-queue.txt [hunk 8]
```diff
@@ -1,103 +1,159 @@
...
+Examples:
+
+.. code:: bash
 
    $ det job update jobID --priority 10
    $ det job update jobID --resource-pool a100
    $ det job update jobID --ahead-of jobID-2
 
-To update a job in batch, provide updates as shown:
+TBD: Is jobID a literal or a variable, which should be shown as <jobID>? Also, need to describe what
```
Comment (eecsliu) [inline docs/features/job-queue.txt:145]: jobID is a variable, and so is jobID-2

File: docs/features/job-queue.txt [hunk 9]
```diff
@@ -1,103 +1,159 @@
...
---help`` for more information. Example operations:
+To use the CLI to modify the job queue, use the ``det job update`` command. For more information,
+run ``det job update --help`` command.
 
-.. code::
+Examples:
+
+.. code:: bash
 
    $ det job update jobID --priority 10
```
Comment (eecsliu) [inline docs/features/job-queue.txt:141]: changes the priority of <jobID> to 10

File: docs/features/job-queue.txt [hunk 10]
```diff
@@ -1,103 +1,159 @@
...
    $ det job update jobID --resource-pool a100
```
Comment (eecsliu) [inline docs/features/job-queue.txt:142]: changes the resource pool of <jobID> to the resource pool named 'a100'.

Also, maybe we should change the name from 'a100' to 'compute-pool' or something...

File: docs/features/job-queue.txt [hunk 11]
```diff
@@ -1,103 +1,159 @@
...
    $ det job update jobID --ahead-of jobID-2
```
Comment (eecsliu) [inline docs/features/job-queue.txt:143]: moves <jobID> ahead of <jobID-2> in the queue. Any currently running jobs lower than and including <jobID-2> may be preempted to run the newly promoted <jobID> job.

**Review by eecsliu — COMMENTED** [review]
Filled in the TBDs and added details where asked. @hamidzr thoughts?

File: docs/features/job-queue.txt [hunk 12]
```diff
@@ -1,103 +1,170 @@
...
++---------------+------------------------------------------------------------------------------+
+| ``Type``      | Job type:                                                                    |
+|               |                                                                              |
+|               | -  ``TYPE_COMMAND``                                                          |
+|               | -  ``TYPE_EXPERIMENT``                                                       |
+|               | -  ``TYPE_NOTEBOOK``                                                         |
+|               | -  ``TYPE_SHELL``                                                            |
+|               | -  ``TYPE_TENSORBOARD``                                                      |
++---------------+------------------------------------------------------------------------------+
+| ``Job Name``  | Command or experiment name, also displayed in the WebUI.                     |
```
Comment (hamidzr) [inline docs/features/job-queue.txt:100]: I think we're going toward calling all of these jobs, at least internally. usage of "command" here could be confusing with command_type command

**Review by hamidzr — APPROVED** [review]
lgtm. sorry for the late review I don't see any open TBDs any more. thank you @eecsliu @gh-determined-ai

**Comment by gh-determined-ai** [comment]
Replaced by PR4151

**Pull request closed without merging**